Source/RapyutaSimulationPlugins/Private/Core/RRUObjectUtils.cpp

```diff
@@ -482,7 +506,7 @@ void URRUObjectUtils::ApplyMaterialProps(UMaterialInstanceDynamic* InMaterial,
     }
 }
 
-bool URRUObjectUtils::SetMeshActorColor(AActor* InMeshActor, const FLinearColor& InColor)
+bool URRUObjectUtils::SetMeshActorColor(AActor* InMeshActor, const FLinearColor& InColor, bool InEmitColor, bool InWhiteMask)
```

Comment:
```suggestion
bool URRUObjectUtils::SetMeshActorColor(AActor* InMeshActor, const FLinearColor& InColor, bool bInEmitColor, bool bInWhiteMask)
```
Also, since the logics for coloring in material is
* Color -> using BlackMask
* Normal (manufacturing texture) -> using WhiteMask
Should we really need `bInWhiteMask`?

Reply:
Using the SetColorMesh() method is more convenient to reset the color as we do not need to create a Material instance before calling the method, also it makes it clear where we are changing the color of the mesh.

Reply:
I see, so for sake of legibility, `bInWhiteMask` may be able to be inferred from `InColor` like
```
bWhiteMask = FLinearColor::Transparent;
```
Also, material instance is always needed beforehand
https://docs.unrealengine.com/5.2/en-US/instanced-materials-in-unreal-engine/#materialinstancedynamic

Reply:
I see, so for sake of legibility, `bInWhiteMask` may be able to be inferred from `InColor` like `bWhiteMask = (FLinearColor::Transparent == InColor);`
Also, material instance is always needed beforehand
https://docs.unrealengine.com/5.2/en-US/instanced-materials-in-unreal-engine/#materialinstancedynamic

Reply:
OK, Removed bInWhiteMask parameter.